pyproject.toml

```
@@ -15,24 +15,43 @@ readme = "README.md"
 
 requires-python = ">=3.10"
 dependencies=[
+  # Hugging Face downloads
   "huggingface_hub",
+
+  # GGUF import
   "gguf",
+
+  # Tiktoken tokenizer for Llama 3 and other advanced models
   "tiktoken",
+
+  # Tokenizers and jinja2 for other non-llama models that use HF tokenizers
   "tokenizers",
   "jinja2",
+
+  # Miscellaneous
   "snakeviz",
   "sentencepiece",
   "numpy>=1.17",
   "blobfile",
   "tomli>=1.1.0; python_version < '3.11'",
```

nit: Can keep the same spacing style

```suggestion
  "tomli>=1.1.0; python_version<'3.11'",
```